File: templates/lang/android_app/src_files/MainActivityTest.java
package {{ .Group }};

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.Robolectric;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.annotation.Config;

import static org.assertj.core.api.Assertions.assertThat;

@RunWith(RobolectricTestRunner.class)
@Config(constants = BuildConfig.class)
public class MainActivityTest {

  @Test
  public void placeholderTest() {
    MainActivity activity = Robolectric.setupActivity(MainActivity.class);

    assertThat(activity.getTitle())
        .isEqualTo("{{ .CamelName }}");
  }
}

File: templates/lang/kotlin_android_app/src_files/MainActivityTest.kt
package {{ .Group }};

import org.assertj.core.api.Assertions.assertThat
import org.junit.Test
import org.junit.runner.RunWith
import org.robolectric.Robolectric
import org.robolectric.RobolectricTestRunner
import org.robolectric.annotation.Config

@RunWith(RobolectricTestRunner::class)
@Config(constants = BuildConfig::class)
class MainActivityTest {

  @Test
  fun placeholderTest() {
    val activity = Robolectric.setupActivity(MainActivity::class.java)

    assertThat(activity.title)
        .isEqualTo("{{ .CamelName }}")
  }
}
